docs-src/typescript/how-to-run-a-worker-on-docker.md

```diff
@@ -9,29 +9,38 @@ tags:
   - typescript
 ---
 
+:::note
+
+Before proceeding, we recommend referring to our [production sample](https://github.com/temporalio/samples-typescript/tree/main/production),
```

Comment:
I'd reverse that sentence. Our recommendation is really that they prepare the workflow bundle ahead-of-time, not that they look at the production sample.

```suggestion
To improve worker startup time, we recommend that workflow bundles be prepared ahead-of-time. See our [production sample](https://github.com/temporalio/samples-typescript/tree/main/production) for details.
```

Reply:
Changed to more or less what you're suggesting.